modules/library/main/src/main/java/org/geotools/styling/FeatureTypeStyle.java

```diff
@@ -136,6 +136,16 @@ public interface FeatureTypeStyle extends org.opengis.style.FeatureTypeStyle {
      */
     public static String SORT_BY_GROUP = "sortByGroup";
 
+    /**
```

Comment:
The flag and its documentation make sense for GeoServer, but in the context of GeoTools they do not (and it's not used anywhere). Maybe best to move this bit to the GeoServer code. Also, features in GeoTools/GeoServer are vector data. Finally, the flag would activate on feature info only. Maybe TRANSFORM_FEATURE_INFO would be a better name.

Reply:
I believe all vendor options can be used in applications built on top of GeoTools, without having to be used in GeoServer to make sense (e.g. a desktop client), and have GeoTools code using them.

This one is a mere constant, with no usage, does not seem to belong here.

Reply:
Would it be better to skip a GeoTools PR and do GeoServer only?

Reply:
Yes, that's what I mean by "Maybe best to move this bit to the GeoServer code"